sdk/src/main/java/io/radar/sdk/RadarGoogleLocationClient.kt

```diff
@@ -0,0 +1,130 @@
+package io.radar.sdk
+
+import android.annotation.SuppressLint
+import android.app.PendingIntent
+import android.content.Context
+import android.location.Location
+import com.google.android.gms.location.*
+
+@SuppressLint("MissingPermission")
+internal class RadarGoogleLocationClient(
+    context: Context,
+    private val logger: RadarLogger
+): RadarAbstractLocationClient() {
+
+    @SuppressLint("VisibleForTests")
+    val locationClient = FusedLocationProviderClient(context)
```

Comment:
would adding a getter remove the need to suppress the lint?

Reply:
This is a property, so there's implicitly a getter. This annotation to stop the linter from complaining that `locationClient` could be `private`. We need to expose it to mock it in tests.